src/test/java/com/spotify/confidence/FeatureProviderTest.java

```diff
@@ -96,19 +97,16 @@ static void after() {
 
   @Test
   public void missingTargetKey() {
```

Comment:
Maybe this test should just be removed? Or reimplemented as a resolve test with missing targeting key. Right now the resolve fails with deadline exceeded, so the test takes some time and it's confusing why it evaluates to an error-case.

Reply:
maybe just removing the targeting key from resolve success test and removing this one?